Tools/clinic/clinic.py

```diff
@@ -5614,6 +5614,10 @@ def state_modulename_name(self, line: str) -> None:
         function_name = fields.pop()
         module, cls = self.clinic._module_and_class(fields)
 
+        if self.kind in {GETTER, SETTER}:
+            if not cls:
+                fail("@getter and @setter must be class methods")
```

Comment:
You mean "instance methods", rather than [class methods](https://docs.python.org/3/howto/descriptor.html#class-methods), right? :)

```suggestion
                fail("@getter and @setter must be instance methods")
```

or

```suggestion
                fail("@getter and @setter must be methods")
```

Reply:
Of course I meant instance methods ... 😅